src/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs

```diff
@@ -396,28 +364,18 @@ public bool Matches(Attribute[] attributes)
         }
 
         /// <internalonly/>
-        int ICollection.Count 
+        bool ICollection.IsSynchronized
         {
```

Comment:
consider adding explicit "private" modifier for readability, here and everywhere else.